lib/smart_city/dataset.ex

```diff
@@ -26,17 +29,19 @@ defmodule SmartCity.Dataset do
       "publishFrequency": "",
       "referenceUrls": [""],
       "rights": "",
+      "riskRating": 0,       // value between 0.0 and 1.0
       "spatial": "",
       "temporal": ""
     },
     "technical": {
+      "allow_duplicates": true
```

Comment:
What is this field for?

Reply:
for reaper to determine if it should de-dupe or not.